setup.py

```diff
@@ -139,56 +148,157 @@ def download_dynamic_library():
+            # Rename <name>.xxx to lib<name>.xxx so the filename is the same for every OS
+            for suffix in suffixes:
+                original_file = os.path.join(arch_dir, "lib", cls.name + suffix)
+                if os.path.exists(original_file):
+                    renamed_file = os.path.join(arch_dir, "lib", "lib" + cls.name + suffix)
+                    os.rename(original_file, renamed_file)
 
             os.remove(filename)
 
+
+class LibDDWaf_Download(Library_Download):
+    name = "ddwaf"
+    download_dir = LIBDDWAF_DOWNLOAD_DIR
+    version = LIBDDWAF_VERSION
+    url_root = "https://github.com/DataDog/libddwaf/releases/download"
+    available_releases = {
+        "Windows": ["win32", "x64"],
+        "Darwin": ["arm64", "x86_64"],
+        "Linux": ["aarch64", "x86_64"],
+    }
+
+    @classmethod
+    def get_package_name(cls, arch, os):
+        archive_dir = "lib%s-%s-%s-%s" % (cls.name, cls.version, os.lower(), arch)
+        return archive_dir
+
+    @classmethod
+    def get_suffixes(cls, os):
+        TRANSLATE_SUFFIX = {"Windows": (".dll",), "Darwin": (".dylib",), "Linux": (".so",)}
+        return TRANSLATE_SUFFIX[os]
+
+    @classmethod
+    def run(cls):
+        LibDDWaf_Download.download_artifacts()
+
+
+class LibDatadog_Download(Library_Download):
+    name = "datadog"
+    download_dir = LIBDATADOG_PROF_DOWNLOAD_DIR
+    version = LIBDATADOG_PROF_VERSION
+    url_root = "https://github.com/DataDog/libdatadog/releases/download"
+    expected_checksums = {
+        "Linux": {
+            "x86_64": "e9ee7172dd7b8f12ff8125e0ee699d01df7698604f64299c4094ae47629ccec1",
+        },
+    }
+    available_releases = {
+        "Windows": [],
+        "Darwin": [],
+        "Linux": ["x86_64"],
+    }
+
+    @classmethod
+    def get_package_name(cls, arch, os):
+        osnames = {
+            "Linux": "unknown-linux-gnu",
+        }
+        tar_osname = osnames[os]
+        archive_dir = "lib%s-%s-%s" % (cls.name, arch, tar_osname)
+        return archive_dir
+
+    @classmethod
+    def get_suffixes(cls, os):
+        TRANSLATE_SUFFIX = {"Windows": (), "Darwin": (), "Linux": (".a", ".h")}
+        return TRANSLATE_SUFFIX[os]
+
+    @staticmethod
+    def get_extra_objects():
+        arch = "x86_64"
+        base_name = "libdatadog_profiling"
+        if CURRENT_OS != "Windows":
+            base_name += ".a"
+        base_path = os.path.join("ddtrace", "internal", "datadog", "profiling", "libdatadog", arch, "lib", base_name)
+        if CURRENT_OS == "Linux":
+            return [base_path]
+        return []
+
+    @staticmethod
+    def get_include_dirs():
+        if CURRENT_OS == "Linux":
+            return [
+                "ddtrace/internal/datadog/profiling/include",
+                "ddtrace/internal/datadog/profiling/libdatadog/x86_64/include",
+            ]
+        return []
+
+    @classmethod
+    def run(cls):
+        LibDatadog_Download.download_artifacts()
```

Comment:
```suggestion
    @classmethod
    def run(cls):
        cls.download_artifacts()
```
it looks like we might be able to put this method in the base class?

Reply:
Indeed, fixed in 08737cc